src/plugins/gr-he-common/core/shell.py

```diff
@@ -103,5 +103,35 @@ def _setup(self):
                 if not self.environment[ohostedcons.CoreEnv.TMUX_PROCEED]:
                     raise context.Abort('Aborted by user')
 
+        if (
+            not self.environment[ohostedcons.NetworkEnv.FORCE_IPV4]
+            and not self.environment[ohostedcons.NetworkEnv.FORCE_IPV6]
+        ):
+            self.logger.warning(
+                _('It has been detected that either --4 or --6 '
+                  'options were not used.\n '
+                  'On a dual stack environment, IPv6 will be selected '
```

Comment:
I'd split the following part to self.dialog.note(). I do not like very long warnings.